services/server/src/server/apiv2/errors.ts

```diff
@@ -0,0 +1,23 @@
+import { NotFoundError } from "../../common/errors";
+import { v4 as uuidv4 } from "uuid";
+
+export type ErrorCode = "unsupported_chain";
```

Comment:
this is wrong right?

Reply:
How do you mean? This is just the first error code that I implemented. There should follow a lot more.

Reply:
So this type should be extended then